src/olympia/constants/base.py

```diff
@@ -222,7 +222,7 @@
 MAX_CATEGORIES = 2
 VALID_CONTRIBUTION_DOMAINS = (
     'donate.mozilla.org', 'micropayment.de', 'patreon.com', 'paypal.com',
-    'paypal.me',
+    'paypal.me', 'opencollective.com'
 )
```

Comment:
We should start sorting this alphabetically, to be honest, otherwise, this looks good.